src/v/config/configuration.cc

```diff
@@ -940,6 +940,18 @@ configuration::configuration()
        .visibility = visibility::user},
       {},
       validate_connection_rate)
+  , kafka_connections_client_group_byte_rate_quota(
+      *this,
+      "kafka_connections_client_group_byte_rate_quota",
+      "Per-group target quota byte rate (bytes per second). "
+      "Client is considered part of the group if client_id prefix "
```

Comment:
should the extraction of the prefix be configurable by a regex? or is there a convention here defined by kafka clients?

Reply:
It is made for Kafka Connect service.
We agreed that groups should be defined by prefixes.
We can change prefixes to regex but I think it doesn't bring any imporvement